lighthouse-cli/index.js

```diff
@@ -139,22 +145,73 @@ if (cli.verbose) {
   flags.logLevel = 'error';
 }
 
-// kick off a lighthouse run
-lighthouse(url, flags, config)
-  .then(results => Printer.write(results, outputMode, outputPath))
-  .then(results => {
-    if (outputMode !== 'html') {
-      Printer.write(results, 'html', './last-run-results.html');
-    }
-    return;
-  })
-  .catch(err => {
-    if (err.code === 'ECONNREFUSED') {
-      console.error('Unable to connect to Chrome. Please run Chrome w/ debugging port 9222 open:');
-      console.error('    npm explore -g lighthouse -- npm run chrome');
-    } else {
-      console.error('Runtime error encountered:', err);
-      console.error(err.stack);
+class Runner {
```

Comment:
really like the simplification you did here.

My only real feedback would be to drop the class and just have the functions be top level, since that's how this is using them anyway (by immediately instantiating the class and calling `run()` on it)